core/test/integration/model/model_posts_spec.js

```diff
@@ -181,16 +180,21 @@ describe('Post Model', function () {
                 });
 
                 it('returns computed fields when columns are asked for explicitly', function (done) {
-                    models.Post.findPage({columns: ['id', 'slug', 'url', 'mobiledoc']}).then(function (results) {
```

Comment:
For this test, just migrated to another computed field as that's what it was meant to check